src/main/java/com/github/charlyb01/music_control/mixin/SoundSystemMixin.java

```diff
@@ -34,7 +34,8 @@ private void delayIfNoSound(CallbackInfo ci) {
     private void printRecord(SoundInstance soundInstance, CallbackInfo ci, WeightedSoundSet weightedSoundSet, Identifier identifier, Sound sound) {
         Identifier record = sound.getIdentifier();
         if (identifier.getPath().contains("music_disc")) {
-            MinecraftClient.getInstance().inGameHud.setRecordPlayingOverlay(Text.translatable(record.toString()));
+            final var mc = MinecraftClient.getInstance();
```

Comment:
Bother you a last time with var. Maybe it's not even useful to make a variable as it's not reused but if you feel like it's clearer you can keep it.